packages/app/styles.css

```diff
@@ -296,3 +270,14 @@ fieldset.field-array fieldset.field-array {
 .modal-sm {
     max-width: 400px;
 }
+
+/* https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html */
+.visually-hidden:not(:focus):not(:active) {
```

Comment:
We're running the version of Bootstrap that doesn't yet include this. In our consideration of mEditor as enterprise software, do you all think we should look into the Horizon Design System and what CSS framework, if any, would help us match it?